src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
I don't know of any other software packages that do this, but I got sick of explaining to beginners why the coefficient changes sign (and label) when you use `contr.sum` instead of the default `contr.treatment` in R. This way makes the coefficients of effects coded predictors parallel the dummy coded versions: coefficients are generated for levels 1:k-1 which represent how much they deviate from some reference (the grand mean or the first level).

My feeling about point 2 there is that it could be very confusing to print a pseudo coefficient whose interpretation is qualitatively different from the other coefficients. I think it would be a good idea to provide some way to generate pseudo coefficients (e.g., full dummy coefficients) from the output (perhaps as an argument to coefs/coeftable). But I haven't thought carefully about how easy this is to do when there are multiple predictors.